x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/components/activity_log_date_range_picker/index.tsx

```diff
@@ -20,9 +20,11 @@ const DatePickerWrapper = styled.div`
   background: white;
 `;
 const StickyFlexItem = styled(EuiFlexItem)`
+  max-width: 350px;
   position: sticky;
   top: ${(props) => props.theme.eui.euiSizeM};
   z-index: 1;
+  padding: ${(props) => `0 ${props.theme.eui.euiFontSizeM}`};
```

Comment:
i wonder if it's possible to use one of the more standard padding sizes like 16px or 24px since it seems kinda weird to use 18px, which looks like its usually used for font sizes

Reply:
@parkiino Yeah, I used 18px as that's what aligns it perfectly with the log content. I'm okay with using a `16px` if we're okay with it being a little off. What do you think?

Reply:
Okay, I used a 16px and it looks alright I think. Let's go with that.

![activity-log-alignment](https://user-images.githubusercontent.com/1849116/129546737-75a794ff-9a7b-4832-99ad-e4dc4335d611.gif)